python/cudf_polars/tests/containers/test_column.py

```diff
@@ -22,6 +22,25 @@ def test_non_scalar_access_raises():
         _ = column.obj_scalar
 
 
+def test_check_sorted():
```

Comment:
This was needed to ensure we hit 100% code coverage with `blocksize-mode="small"`.